From 46c44901e63c48e9ebd37fa02daa0fcda57f8fc5 Mon Sep 17 00:00:00 2001
From: Shadi Ammouri <shadi@marvell.com>
Date: Wed, 30 Mar 2016 13:55:12 +0300
Subject: [PATCH 0284/2241] stm: Temporarily disable coresight trace for AP806

This reverses commit 30ce52c54be5 ("coresight: add linux drivers for
coresight trace")',
till this feature is tested to be working on AP806-A0.

Change-Id: I36f636b4ad17d9999b05c5dde564d757866a7282
Signed-off-by: Shadi Ammouri <shadi@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/28667
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Yehuda Yitschak <yehuday@marvell.com>
---
 arch/arm64/configs/mvebu_v8_lsp_defconfig | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/arch/arm64/configs/mvebu_v8_lsp_defconfig b/arch/arm64/configs/mvebu_v8_lsp_defconfig
index c2f3736..472c317 100644
--- a/arch/arm64/configs/mvebu_v8_lsp_defconfig
+++ b/arch/arm64/configs/mvebu_v8_lsp_defconfig
@@ -197,7 +197,6 @@ CONFIG_CLK_VEXPRESS_OSC=y
 CONFIG_ARM_SMMU=y
 CONFIG_RESET_CONTROLLER=y
 CONFIG_PHY_XGENE=y
-CONFIG_STM=y
 CONFIG_EXT2_FS=y
 CONFIG_EXT3_FS=y
 CONFIG_BTRFS_FS=y
@@ -231,9 +230,6 @@ CONFIG_LOCKUP_DETECTOR=y
 # CONFIG_DEBUG_PREEMPT is not set
 # CONFIG_FTRACE is not set
 CONFIG_MEMTEST=y
-CONFIG_CORESIGHT=y
-CONFIG_CORESIGHT_LINK_AND_SINK_TMC=y
-CONFIG_CORESIGHT_SOURCE_ETM4X=y
 CONFIG_SECURITY=y
 CONFIG_CRYPTO_ANSI_CPRNG=y
 CONFIG_ARM64_CRYPTO=y
-- 
2.7.4

